src/app/container/tool-file-editor/tool-file-editor.component.ts

```diff
@@ -114,7 +114,13 @@ export class ToolFileEditorComponent extends FileEditing {
           this.toggleEdit();
           this.containerService.setTool(editedDockstoreTool);
           const updatedVersion = this.getNewestVersion(editedDockstoreTool.workflowVersions);
-          this.alertService.detailedSuccess('Saved version ' + updatedVersion.name + ' of hosted entry ' + editedDockstoreTool.namespace);
+          this.alertService.detailedSuccess(
+            'Saved version ' +
+              updatedVersion.name +
+              ' of hosted tool ' +
+              editedDockstoreTool.name +
+              (editedDockstoreTool.toolname ? '/' + editedDockstoreTool.toolname : null)
```

Comment:
I believe this line will add the string 'null' to the end of the message. Javascript is funky and will convert null and undefined to strings when you try and concatenate them with a string variable, so:
<img width="325" alt="Screen Shot 2020-12-11 at 2 54 22 PM" src="https://user-images.githubusercontent.com/36607471/101948892-cdf3d200-3bc0-11eb-89a1-172a0135dbaa.png">


You could replace ```null``` with an empty string, ```""```, which would work as intended.